src/main/scala/com/github/xiaodongw/swagger/finatra/FinatraOperation.scala

```diff
@@ -92,7 +92,12 @@ class FinatraOperation(operation: Operation) {
     val model = schema match {
       case null => null
       case p: RefProperty => new RefModel(p.getSimpleRef)
-      case _ => null  //todo map ArrayProperty to ArrayModel?
+      case p: ArrayProperty => {
```

Comment:
I'd probably make a schema util or something, and just centralize this and the other match statements there so we dont have code duplication